src/translations/index.js

```diff
@@ -35,19 +35,21 @@ const TRANSLATIONS = {
         'import-heading-protect': 'Protect your wallet',
         'import-text-set-password': 'You can now set a password to encrypt your wallet on this device.',
         'import-upload-login-file': 'Upload your Login File',
+        'import-unlock-account': 'Unlock your account',
         'import-create-account': 'Create new Account',
 
         'import-file-lost-file': 'Lost your Key File? You can recover your account with your 24 Recovery Words.',
         'import-file-button-words': 'Login with Recovery Words',
         'import-file-heading-unlock': 'Unlock your Key File',
```

Comment:
```suggestion
        'import-file-heading-unlock': 'Unlock your Login File',
```
Potentially unused. Still i just saw it there.
However there are more of them left in `translations/index.js`, so probably better to do in a separate wording PR.